tests/test_ic_od_to_kvp_wrapper/test_od_as_kvp.py

```diff
@@ -0,0 +1,38 @@
+import unittest
+
```

Comment:
could you rename to test_object_detection_as_kvp.py for consistency?

Reply:
updated